### PR TITLE
Multisig improvements

### LINKED ISSUE #175: Feature: add multisig account management
Kill manage and convert to form with manage
image

### LINKED ISSUE #318: UI: Display multisig account info
We can make a tiny component that should be shown in /multisig, and could be used somewhere else as well

### PR DESCRIPTION
**Pull request by decentraliser**
- Add multisig account management, fixes #175
- Add a multisig info component in multisig transaction form, fixes #318
- Auto reload multisig info when a multisig modification transaction is confirmed

@domakun I let my first proposition for tree update because when an account's data is refreshed, it collapses all the nodes, didn't find a fix to that yet and moved on...

### CODE REVIEW

**Comment by NemTechCN** [comment]
image
https://github.com/nemfoundation/nem2-desktop-wallet/issues/175
I think we need to kill convert or manage panel .

File: src/language/jp-JP.ts [hunk 1]
```diff
@@ -499,7 +499,7 @@ export const jp_JP = {
     Convert_to_multi_sign_account: 'マルチシグアカウントへ変換',
     cosigner: '連署者',
     Add_co_signers_here_will_be_displayed_in_the_action_list_click_delete_to_cancel_the_operation: '共同署名者を追加します。ここで操作リストに表示されます。削除をクリックして取消操作ができます',
-    Wallet_account_address_or_alias: 'ウォレットアドレスまたはエイリアス',
```
Comment (NemTechCN) [inline src/language/jp-JP.ts]: We can move the updated sentence at the end of file , so people who translate Japanese can clearly see this sentence

Comment (decentraliser) [reply src/language/jp-JP.ts]: > 1. kill convert panel ;
> 2. put updated Japanese sentence at end of file

OK for Japanese

However we cannot kill one of the tab

If account A had no cosignatory, but has multisig accounts, then you want both tabs

Comment (NemTechCN) [reply src/language/jp-JP.ts]: Manage and Covert similarity is higher than 50% , we can merge them by adding a selection to choose initiator like this 
image

**Comment by decentraliser** [comment]
Updated, IMO we cannot kill one of the tab

If account A has no cosignatory, but has multisig accounts, then you want both tabs
(can convert A, can manage A's multisig accounts)

**Pull request merged**